tests/test_inference.py

```diff
@@ -218,13 +218,67 @@ def test_bootstrap_testset_rdm(self):
         evaluations, n_rdms = bootstrap_testset_rdm([m, m2], rdms,
             method='cosine', fitter=None, N=100, rdm_descriptor=None)
 
-class test_save_load_result(unittest.TestCase):
+class test_save_load(unittest.TestCase):
```

Comment:
Class names by convention are written in *CamelCase*

Reply:
Thanks for the reminder!